src/main/java/hu/progmasters/probac/Morse.java

```diff
@@ -0,0 +1,85 @@
+package hu.progmasters.probac;
+
+/**
+ * Converts string into Morse Code
+ */
+public class Morse {
+    /**
+     * Starts main code
+     * @param args String array standard start parameter
+     */
+    public static void main(String[] args) {
+        Morse morse = new Morse();
+        morse.printMorseCode("Wozu Worte drucken, es gibt doch Schreiber");
+    }
+
+    /**
+     * Returns index of letter in alphabet.
+     * @param letter String whose index is to be found in alphabet.
+     * @return index of letter
+     */
+    int getPosInAlphabet(String letter) {
+        letter = letter.toLowerCase();
+        String alphabet = "abcdefghijklmnopqrstuvwxyz1234567890";
+        return alphabet.indexOf(letter);
+    }
+
+    /**
+     * prints String converted into Morse code
+     * @param input String that is to be converted
+     */
+    void printMorseCode(String input) {
```

Comment:
Please implement (extract from printMorseCode() Method) a 
public void ConvertText2Morse(String input, List<String> reply, String[] morsecode);
Method.
This Method should be called from the printMorseCode() Method

Reply:
CpnvertText2Morse was extracted, please review if the new method is satisfactory!

Reply:
ConvertText2Morse was further tweaked, now it contains code responsible for converting input to lowercase as well as an array functioning as an alphabet-Morse dictionary.

Reply:
Looks like using a few lines of input.replace instead of a single replace.All with specific excpetions was the better way to go after all. ConvertText2Morse was modified accordingly, now all current tests of it run without issue.